molecule/default/prepare.yml

```diff
@@ -1,15 +1,10 @@
 ---
 
-- name: Install depends
+- name: Install dependencies
   hosts: all
   gather_facts: no
   tasks:
     - name: install apt depends
       apt:
-        name: "{{item}}"
+        name: ["net-tools", "lsb-release", "curl", "procps"]
```

Comment:
Is there any issue with the previous format ?

Reply:
yes, this one:

[DEPRECATION WARNING]: Invoking "apt" only once while using a loop via
squash_actions is deprecated. Instead of using a loop to supply multiple items
and specifying name: "{{item}}", please use name: ['package', 'package'] and remove the loop.

Reply:
hum ok, is this kind of format work ? (same question, this is working with ansible 2.4 right ?)

```
    - name: install apt depends
      apt:
        name: "{{item}}"
        name:
            - net-tools
            - lsb-release
            - ...
```

Reply:
Yes it works, but you will have the deprecation warning. I don't remember until which version.

After fixing it and before push it I tested it locally and then with the pipeline with the following ansible versions: 2.4*, 2.5.*, 2.6.*, 2.7.*

Reply:
let me see, I supposed with: **with_items**

Reply:
I updated the example, `        name: "{{item}}"` was a copy paste error

Reply:
it works, do you prefer this format?

Reply:
yep, easier to read and add new content in it.
From the code or github diff

Reply:
done